frontends/beta-frontend/shared/components/Grid/TeamColumn.tsx

```diff
@@ -25,65 +25,125 @@ export const teamColumn = [
   { field: 'age', headerName: 'Age', headerAlign: 'center' as GridAlignment, align: 'center' as GridAlignment, minWidth: 70, flex: 1 },
   {
     field: 'position',
+    valueFormatter: (params: GridValueFormatterParams<any>) => {
+      switch (params.value) {
+        case 'GOALKEEPER':
+          return 'GK';
+        case 'DEFENDER':
+          return 'DF';
+        case 'MIDDLE':
+          return 'MD';
+        case 'FORWARD':
+          return 'FW';
+        default:
+          return params.value;
+      }
+    },
     headerName: 'Position',
     headerAlign: 'center' as GridAlignment,
     align: 'center' as GridAlignment,
     minWidth: 70,
     flex: 1,
+    // Assign Css class based on position
+    cellClassName: (params: GridCellParams<string>) => {
+      if (params.value == null) {
+        return '';
+      }
+
+      return clsx('super-app', {
+        [params.value.toLowerCase()]: true,
+      });
+    },
   },
   {
     field: 'DEFENSE_POSITION',
-    headerName: 'DP',
+    headerName: (<div>DP<sup>CO</sup></div>),
     minWidth: 50,
     headerAlign: 'center' as GridAlignment,
     align: 'center' as GridAlignment,
     flex: 1,
-    valueGetter: (params: GridValueGetterParams) => params.row.actualSkills.DEFENSE_POSITION,
+    valueGetter: (params: GridValueGetterParams) => {
+      if (params.row.actualSkills.DEFENSE_POSITION) {
```

Comment:
if you would like to shorten this code you could use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR#short-circuit_evaluation
so this code should look like (notice ? after DEFENSE_POSITION)
`return params.row.actualSkills.DEFENSE_POSITION?.PlayerSkills.actual || params.row.actualSkills.CONTROL.PlayerSkills.actual;`